### ISSUE TITLE
Poor data quality: STIR

### ISSUE THREAD

**Issue by valosekj** [ivadomed/canproco#22]
- `sub-cal149_ses-M0_STIR.nii.gz` - FOV does not cover the whole SC in sagittal plane:
<img width="1014" alt="image" src="https://user-images.githubusercontent.com/39456460/216417747-5add675f-d2a3-4fab-a571-91cc0bb4f445.png">

**Comment by valosekj** [comment]
TODO: add those subjects to [etc/exclude.yml](https://github.com/ivadomed/canproco/blob/main/etc/exclude.yml)

**Comment by Nilser3** [comment]
@valosekj 
I have found some MS studies that have a _lesion-manual.nii.gz_ file, however these are empty (could be cases of a non-lesion patient). Would it be good to identify them so that they do not participate in training?

**Comment by valosekj** [comment]
> I have found some MS studies that have a _lesion-manual.nii.gz_ file, however these are empty (could be cases of a non-lesion patient).

Good point! IIRC, there are indeed some MS patients with no lesions. To keep track of this, we created an empty `lesion-manual.nii.gz` mask (i.e., only zeros).

>  Would it be good to identify them so that they do not participate in training?

Good question! In [model_seg_sci](https://github.com/ivadomed/model_seg_sci) project, we use all subjects (i.e., even those with empty lesion masks) to teach the model even such cases.

**Comment by jcohenadad** [comment]
> Would it be good to identify them so that they do not participate in training?

As @valosekj mentioned, it is important for the model to learn true negatives. However, it would be good to double check how the loss is computed in this case during training (ie: if there is no segmentation, there is no Dice score 😉, unless if the background is taken into account, but in the latter case that would provoke strong class imbalance).

**Comment by plbenveniste** [comment]
@valosekj 
I think there is a mistake here: `sub-mon006` has only a PSIR file. 

- [Referencing this in [issue 53](https://github.com/ivadomed/canproco/issues/53) which deals with data quality for PSIR contrast. 
- Modifying `exclude.yml` file under right contrast category

**Comment by valosekj** [comment]
> @valosekj I think there is a mistake here: `sub-mon006` has only a PSIR file.
> 
> * [Referencing this in [issue 53](https://github.com/ivadomed/canproco/issues/53) which deals with data quality for PSIR contrast.
> * Modifying `exclude.yml` file under right contrast category

Correct. Good catch! Thanks!
